lib/active_model_serializers/cached_serializer.rb

```diff
@@ -0,0 +1,42 @@
+require 'active_model_serializers/fragment_cache'
+module ActiveModelSerializers
+  class CachedSerializer
+    def initialize(serializer)
+      @cached_serializer = serializer
+      @klass             = @cached_serializer.class
```

Comment:
I think using `serializer.class` here is less typo-prone. A typo on `serializer` would result in `NameError` whereas a typo in `@cached_serializer` would result in `nil`.